### PR TITLE
[FE] fix: 아이패드 환경 스크롤 대처

### LINKED ISSUE #780: [FE] bug: 전화번호 입력 페이지 스크롤을 막는다.
## 버그설명

태블릿 모드에서의 매끄러운 터치를 위함입니다. 

<br/>

## 체크리스트

- [x] `assignee` 설정 (선택)
- [x] `labels` 설정
- [x] `milestone` 설정

<br/>

### PR DESCRIPTION
**Pull request by kangyeongmin**
## 관련 이슈

closes #780

## 체크리스트

- [x] `reviewers` 설정
- [x] `label` 설정
- [x] `milestone` 설정

### CODE REVIEW

**Review by 2yunseong — APPROVED** [review]
image
👍 👍

File: frontend/src/pages/Admin/EnterPhoneNumber/style.tsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
 export const Title = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: center;
   position: relative;
-  width: 100vw;
-  height: 110px;
+  width: 100%;
+  height: 80px;
```
Comment (regularPark) [inline frontend/src/pages/Admin/EnterPhoneNumber/style.tsx:9]: what's difference between `vw` and `%`?

**Review by regularPark — APPROVED** [review]
wow! amazing responsive screen for tablet!

File: frontend/src/pages/Admin/EnterPhoneNumber/style.tsx (hunk 1 above)
Comment (2yunseong) [reply frontend/src/pages/Admin/EnterPhoneNumber/style.tsx:9]: i think `vw` based on browser size but `%` based on parent element size.

Comment (kangyeongmin) [reply frontend/src/pages/Admin/EnterPhoneNumber/style.tsx:9]: ㄴ It's correct. That's what i want to say.

**Pull request merged**